### ISSUE TITLE
Modify number of images when loading a new config

### ISSUE THREAD

**Issue by psavery** [HEXRD/hexrdgui#244]
Currently, if the user tries to load a new config while there are already images loaded, and the number of detectors in the new config doesn't match the number of images already loaded, an error is produced.

Rather than producing an error, we should modify the number of images so that it matches. Perhaps just removing all images will also be a good solution (since, the mismatch in the number of detectors and the number of images most likely means the config isn't for those images).

**Comment by bnmajor** [comment]
#256 removes any loaded images when a new configuration file is loaded. Currently this happens for every new config file. Should this only happen when the new config has a different number of detectors and/or different detector names than the currently loaded config file?

**Comment by psavery** [comment]
I think it might be good to check the detector names before/after to see if they match (so something like `prev_detectors = HexrdConfig().get_detector_names()`, then after the load, `new_detectors = HexrdConfig().get_detector_names()`, then check):

```python
    if prev_detectors != new_detectors:
        # Clear the images
```

Just because someone might be comparing different configs of the same detector, and this would allow them to not have to re-load the images each time. But if that's not a likely scenario, then maybe it doesn't matter.

**Comment by bnmajor** [comment]
I've added in a check to see if the new detectors differ from the old before clearing the images. Additionally, the loaded images are now also cleared if a detector is manually added or removed using the `Instrument Calibration` panel, under the `Form View` tab.

**Comment by psavery** [comment]
Fixed by #256
#260 Will also make it so dummy images are loaded when a new config is loaded.
